include/nuttx/sched.h

```diff
@@ -72,6 +72,18 @@
 #  define CONFIG_SCHED_SPORADIC_MAXREPL 3
 #endif
 
+#ifndef CONFIG_SCHED_CRITMONITOR_MAXTIME_THREAD
+#  define CONFIG_SCHED_CRITMONITOR_MAXTIME_THREAD -1
```

Comment:
These symbols are only used in sched_critmonitor.c, why are there these errors? You need to align your code with the mainline:
https://github.com/apache/nuttx/blob/master/sched/sched/sched_critmonitor.c#L41-L51